public/locales/en/common.json

```diff
@@ -134,13 +134,16 @@
       "LESSON_PART": "Daily Lessons",
       "DAILY_LESSON": "Daily Lessons",
       "VIRTUAL_LESSON": "Webinars",
+      "VIRTUAL_LESSONS": "Webinars",
       "LECTURE": "Lectures",
       "WOMEN_LESSON": "Women Lessons",
       "CHILDREN_LESSON": "Children Lessons",
       "LESSONS_SERIES": "Study Series",
       "LECTURE_SERIES": "Lecture Series",
       "CLIP": "Clips",
+      "CLIPS": "Clips",
```

Comment:
What's difference between CLIP and CLIPS?

Reply:
same as VIRTUAL_LESSONs